Dart/src/com/jetbrains/lang/dart/ide/formatter/DartBlock.java

```diff
@@ -184,4 +221,16 @@ public BlockWithParent getParent() {
   public void setParent(BlockWithParent newParent) {
     myParent = newParent;
   }
-}
\ No newline at end of file
+
+  private static class DartCascadeBlock extends DartBlock {
```

Comment:
Obviously, I disagree, but I'm not trying to teach you good programming style. It's a moot point since the code has been deleted.